ui/src/pages/downloads.tsx

```diff
@@ -0,0 +1,89 @@
+import { IconDefinition, faApple, faLinux, faWindows } from '@fortawesome/free-brands-svg-icons';
+import { faCircleDown, faCodeCommit, faExternalLink } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+import { PageTitle } from '@ui/features/common';
+
+const DownloadLink = ({ url, children }: { url: string; children: React.ReactNode }) => (
+  <a
+    href={url}
+    target='_blank'
+    rel='noreferrer'
+    className='bg-blue-500 inline-flex items-center text-white px-2 py-1 rounded text-sm font-medium no-underline'
+  >
+    <FontAwesomeIcon icon={faCircleDown} className='mr-2' />
+    {children}
+  </a>
+);
+
+const DownloadItem = ({
+  title,
+  icon,
+  links
+}: {
+  title: string;
+  icon: IconDefinition;
+  links: { url: string; title?: string }[];
+}) => (
+  <div className='p-4 rounded bg-neutral-100 flex flex-col' style={{ width: '250px' }}>
+    <div className='flex items-center mb-4 text-neutral-800 font-medium text-lg mx-auto'>
+      <FontAwesomeIcon icon={icon} className='mr-2' />
+      {title}
+    </div>
+    <div className='flex items-center gap-2 justify-center'>
+      {links.map((link, index) => (
+        <DownloadLink key={index} url={link.url}>
+          {link.title || 'Download'}
+        </DownloadLink>
+      ))}
+    </div>
+  </div>
+);
+
+const constructUrl = (platform: string, version?: string) =>
+  `https://github.com/akuity/kargo/releases/${version ? '' : 'latest/'}download/${version ? `${version}/` : ''}${platform}`;
+
+export const Downloads = () => (
+  <div className='p-6'>
+    <PageTitle title='CLI Downloads' />
+    <div className='text-2xl mb-2 font-semibold flex items-center'>
+      <FontAwesomeIcon icon={faCodeCommit} className='mr-2' />
+      {__UI_VERSION__ === 'development' ? 'Latest version' : __UI_VERSION__}
+    </div>
+    <a
+      href='https://github.com/akuity/kargo/releases'
+      target='_blank'
+      rel='noreferrer'
+      className='mb-6 flex text-xs items-center text-blue-500'
+    >
+      <FontAwesomeIcon icon={faExternalLink} className='mr-2' />
+      VIEW ALL RELEASES
```

Comment:
I have been wondering for some time why these types of things use literal capslock, instead of something like `text-transform: capitalize`.

Reply:
Good question. The reason to use caps lock vs. text transform is for accessibility: if someone uses a screen reader, for example, it will typically not detect the text transform, but if the text is _actually_ capitalized, it will correctly indicate that the text is all-caps.

Reply:
I was always under the impression this would do the opposite thing for most screen readers, as it will, for example, read certain bits as acronyms (e.g. "IT" becomes "I.T.").

See also:

- https://www.timdunklee.com/notes/use-css-text-transform-for-uppercase-letters-a11y/
- https://www.stefanjudis.com/today-i-learned/text-transforms-affects-screen-readers-too/

Reply:
TIL! Thanks for the links!